assets/sass/base/_mixins.scss

```diff
@@ -0,0 +1,74 @@
+// Breakpoint media query mixin
+// 
+// Our custom breakpoint mixin since Foundations @media functions only allow defined Foundation size variables.  
+// Sometimes you just need a breakpoint at a specific size, which is what this mixin allows you to do.  Enjoy!
+// $breakpoint-default set in /assets/sass/base/_variables to change default
+// 
+// @param {string} $breakpoint - breakpoint value measured in any unit
+// @param {string} $size - width specification, value is 'min' or 'max'
+@mixin breakpoint( $breakpoint: $breakpoint-default, $size: min ) {
```

Comment:
Is the $breakpoint-default a var that sets a default value so a user doesn't have to keep repeating a common value?  I think you mentioned this before....just clarifying.